torch/_inductor/config.py

```diff
@@ -479,6 +479,11 @@ class cuda:
     # 4) default system search PATH.
     cuda_cxx = None
 
+    # If set to True, it will ensure that only GEMM ops capable of
+    # epilogue fusion via CUTLASS Epilogue Visitor Trees ( EVT )
+    # are enabled for the CUTLASS backend.
+    cutlass_only_evt_capable_ops: bool = False
```

Comment:
New config setting, used by the unit test to ensure that only GEMM ops with EVT may be picked